solidity/test/token_grant/TestTokenGrantStake.js

```diff
@@ -1,6 +1,4 @@
-
```

Comment:
All `increaseTime` values look good to me but shouldn't we use `increaseTimeTo` in this file as well?